### ISSUE TITLE
check_crls: false in receiver config - investigate if this should be true

### ISSUE THREAD

**Issue by tofu-rocketry** [apel/ssm#276]

**Comment by RedProkofiev** [comment]
> -crl_check_all
> Checks the validity of **all** certificates in the chain by attempting to look up valid CRLs.
Yes, it does.  We can add/replace it with the option of just -crl_check to only check the first certificate.
Sourced from https://www.openssl.org[/docs/man1.0.2/man1/openssl-verify.html](https://www.openssl.org/docs/man1.0.2/man1/openssl-verify.html)
